UPDATING.md

```diff
@@ -0,0 +1,22 @@
+# Upgrade / Migration Guide
+
+## Version 1.1.8 to 1.2.0
+
+### Notable Changes
+This release is all about channel options. Here is the full [changelog](https://github.com/ably/ably-ruby/blob/main/CHANGELOG.md)
+
+* Channel options were extracted into a seperate model [ChannelOptions](https://github.com/ably/ably-ruby/blob/main/lib/ably/models/channel_options.rb). However it's still backward campatible with `Hash` and you don't need to do make any adjusments in your code
+
+* `ChannelOptions` now support `:params`, `:modes` and `:cipher` as options. Previosly only `:cipher` was available
```

Comment:
I'd prefer "The `ChannelOptions` class now supports `:params`, `:modes` and `:cipher` as options.".

Also Previosly -> Previously